src/styles/blocks/price-box.scss

```diff
@@ -0,0 +1,17 @@
+@import "../utils/variables";
+
+.price-box {
```

Comment:
You have styles related to `price-box` in two places, one for class `card__price-box` and another one here, pick one class name and keep all styles for this component in the same place.